README.md

```diff
@@ -2,6 +2,12 @@
 
 Ember shim for loading `d3@4.x.x`. To install:
 
+First, make sure you're running on npm@3. If it's not the case then run
```

Comment:
We should probably specify that you run `npm -v` to find out. Also, npm 4 is out now, so maybe just say "npm > 2"